README.md

```diff
@@ -172,7 +172,9 @@ resolved, and other properties, as they are determined.
     integrity signature of a manifest, if present.  There must be a
     configured `_keys` entry in the config that is scoped to the
     registry the manifest is being fetched from.
-
+* `verifyAttestations` A boolean that will make pacote verify Sigstore
```

Comment:
Is there a compelling reason why this is a separate flag than the one we use to validate signatures?

Reply:
> Is there a compelling reason why this is a separate flag than the one we use to validate signatures?

Not wedded to this but was thinking if we want to allow opting out of one but not the other check in the CLI. The other thought I had was that we could roll this out without enabling it immediately in the CLI.

Thoughts?

Reply:
Not rolling it out by default in the CLI seems like a compelling reason!  Thanks.